lib/measures/urban_geometry_creation/measure.rb

```diff
@@ -789,45 +165,44 @@ def run(model, runner, user_arguments)
     end
 
     @origin_lat_lon = OpenStudio::PointLatLon.new(min_lat, min_lon, 0)
-    
+
     site = model.getSite
     site.setLatitude(@origin_lat_lon.lat)
     site.setLongitude(@origin_lat_lon.lon)
-    
-    if feature[:properties][:surface_elevation]
-      surface_elevation = feature[:properties][:surface_elevation].to_f
+
+    if feature.get(:surface_elevation)
```

Comment:
I can see a few options here:

```
feature.name
feature[:name]
feature.get(:name)
```

I like the first one because it shows up in rdoc, second one is natural for accessing a hash, i'm not as crazy about the third

Reply:
`feature` is no longer a geojson object, it's an instance of the feature class. I think that first option would translate into `feature.feature_json[:properties].name` ?  Would you still prefer that over the instance method `get(name)`? :)

Reply:
For the second option you could implement the `[]` method in the Feature class like https://stackoverflow.com/questions/10018900/how-does-defining-square-bracket-method-in-ruby-work

You can also override the `method_missing` behavior to check for a key in the hash like https://stackoverflow.com/questions/5764454/hash-keys-as-accessors-in-a-class

Personally I like the first option best